src/write/statistics.rs

```diff
@@ -57,7 +57,11 @@ pub fn reduce(stats: &[&Option<Arc<dyn Statistics>>]) -> Result<Option<Arc<dyn S
         PhysicalType::Int64 => {
             let stats = stats.iter().map(|x| x.as_any().downcast_ref().unwrap());
             Some(Arc::new(reduce_primitive::<i64, _>(stats)))
-        }
+        },
+        PhysicalType::Int96 => {
+            let stats = stats.iter().map(|x| x.as_any().downcast_ref().unwrap());
+            Some(Arc::new(reduce_primitive::<[u32; 3], _>(stats)))
```

Comment:
is the order of [u32; 3] the same as its corresponding representation of a timestamp? I.e. don't we need to transform it and then apply the reduce?

Reply:
I'm assuming reading is handled before we get to fold:
`deserialize_statistics` -> `primitive::read::<[u32; 3]>` -> `types::decode(x)` -> `T::from_le_bytes`.
Then, once we've got `NativeType<[u32; 3]>` we can use ord it implements (` int96_to_i64_ns(*self).ord(&int96_to_i64_ns(*other))`). 

Am I missing something?